receiver/otlpjsonfilereceiver/file.go

```diff
@@ -74,14 +77,33 @@ func (f *receiver) Shutdown(ctx context.Context) error {
 	return f.input.Stop()
 }
 
+func (f *receiver) storageClient(ctx context.Context, host component.Host) (storage.Client, error) {
```

Comment:
Is this a duplicate of the code in `pkg/stanza/adapter/storage.go`?  Is there a way to reuse?

Reply:
I pulled it back into one place. Longer term, I think it belongs as part of [this proposal](https://github.com/open-telemetry/opentelemetry-collector/issues/5832).